tests/test_kubernetes_tools.py

```diff
@@ -3144,10 +3145,22 @@ def test_get_active_shas_for_service():
                 }
             )
         ),
+        mock.Mock(
+            metadata=mock.Mock(
+                labels={
+                    "yelp.com/paasta_config_sha": "c123",
+                    "yelp.com/paasta_git_sha": "d456",
+                    "paasta.yelp.com/config_sha": "c123",
```

Comment:
What's the difference between the `yelp.com/` and the `paasta.yelp.com/` labels? I think we've only added the `image_version` to the paasta ones. Should I change that?

Reply:
I believe we can safely just add `paasta.yelp.com` versions for any new labels that are specific to paasta workloads.  The `yelp.com/paasta_*` versions are there for legacy reasons, and there were some attempts in the past to move away from them afaik